login/login.ftl

```diff
@@ -85,8 +85,8 @@
             </div>
 	    </#if>
       </div>
-      </div>
-    <p>${msg("footer")?no_esc}</p>
+
+    <div class="col-sm-10 col-sm-offset-1 col-md-6 col-md-offset-3">${msg("footer")?no_esc}</div>
```

Comment:
Will this work with the split view for CILogon and where CILogon isn't enabled? I presume that by taking up all 12 columns in all views this will always go under the login form(s).

Reply:
The totalsim is the only thing that specifies the footer message in the messages.properties file `${msg("footer")?no_esc}`.

Currently there is an empty `<p></p>` on the CILogon login page with the same stray `</div>` when showing the split view. If we did set the same footer in the message.properties it wouldn't look pretty on that page but would still be functional:

![screen 2019-10-08 at 5 06 36 PM](https://user-images.githubusercontent.com/512333/66433346-298b6400-e9ee-11e9-8eba-228b8ec3f9ab.png)

Reply:
At least that's what I think. I haven't actually loaded these changes into a Keycloak instance to confirm. I'm not sure what the development process is for themes now.

Reply:
I have been modifying the *-test themes on webdev06 which is what the realms use on that server. The main theme ending in -test is in git, but permissions maybe off.  I say merge and tag as bug fix then we can push to test and see what it looks like.

Look in /opt/keycloak/themes on webdev06 and there is ood-keycloak-test , osc-test and awesim-test. Those are what I've modifed where files from ood-keycloak-test are copied to osc-test or awesim-test. I tried to set my umask such that others could access the files but really need to come up with better method.